### ISSUE TITLE
Scrolling results using cursor keys incorrect, highlighted item off screen.

### ISSUE THREAD

**Issue by phhowe17** [qarmin/czkawka#803]
v5.0.0

When using similar image function, using the keyboard up/down errors to scroll highlighted item, the windows fails to scroll up to keep the highlighted item in the display window.  e.g. using down arrow to move selection will move off screen instead of scrolling the file list upward to keep the current item visible.

**Comment by EvocativeOlash** [comment]
Yep persistant issue, very fatiguing on long lists of similar duplicates.

**Comment by qarmin** [comment]
I can't reproduce with gtk frontend, but can with Krokiet

**Comment by qarmin** [comment]
Krokiet version should no longer have such problem
